content/docs/user-guide/plots.md

```diff
@@ -0,0 +1,420 @@
+  range
+
+Note that in the case of CSV/TSV metrics files, column names from the table
+header (first row) are equivalent to field names.
+
+### Custom templates
+
+Plot templates are [Vega-Lite](https://vega.github.io/vega-lite/) JSON
+specifications. They use predefined DVC anchors as placeholders for DVC to
+inject the plot values.
+
+- `<DVC_METRIC_DATA>` (**required**) - the plot data from any type of metrics
+  files is converted to a single JSON array, and injected instead of this
+  anchor. Two additional fields will be added: `step` and `rev` (explained
+  below).
+
+- `<DVC_METRIC_TITLE>` (optional) - a title for the plot, that can be defined
+  with the `--title` option of the `dvc plots` subcommands.
+
+- `<DVC_METRIC_X>` (optional) - field name of the data for the X axis. It can be
+  defined with the `-x` option of the `dvc plots` subcommands. The
+  auto-generated `step` field (explained below) is the default.
+
+- `<DVC_METRIC_Y>` (optional) - field name of the data for the Y axis. It can be
+  defined with the `-y` option of the `dvc plots` subcommands. It defaults to
+  the last header of the metrics file: the last column for CSV/TSV, or the last
+  field for JSON/YAML.
+
+- `<DVC_METRIC_X_LABEL>` (optional) - field name to display as the X axis label
+
+- `<DVC_METRIC_Y_LABEL>` (optional) - field name to display as the Y axis label
+
+<details>
+
+### Expand to learn how DVC modifies plot data for rendering.
+
+File targets given to `dvc plots show` and `dvc plots diff` are treated as
+separate data series, each to be injected into a template file. There are two
+important fields that DVC adds to the plot data:
+
+- `step` - zero-based counter for the data rows/values. In many cases it
+  corresponds to a machine learning training epoch number.
+
+- `rev` - This field helps distinguish between data sourced from different
+  revisions, files or columns.
+
+</details>
+
+Refer to [`templates`](/doc/command-reference/plots/templates) command for more
+information on how to prepare your own template from pre-defined ones.
+
+## Custom HTML templates
+
+It's possible to supply an HTML file to `dvc plots show` and `dvc plots diff` by
+using the the `--html-template` option. This allows you to customize the
+container where DVC will inject plots it generates.
+
+> ⚠️ This is a separate feature from
+> [custom Vega-Lite templates](/doc/user-guide/plots#custom-templates).
+
+The only requirement for this HTML file is to specify the place to inject plots
+with a `{plot_divs}` marker. See an
+[example](/doc/user-guide/plots#example-offline-html-template) that uses this
+feature to render DVC plots without an Internet connection, below.
+
+## Example: Offline HTML Template
+
+The plots generated by `dvc plots` uses Vega-Lite JavaScript libraries, and by
+default these load [online resources](https://vega.github.io/vega/usage/#embed).
+There may be times when you need to produce plots without Internet access, or
```

Comment:
... In the user guide we don't usually have separate examples like these. We embed them into the explanations instead. Think book format.

Reply:
> all the plot template info is in the new guide now

... ideally we would dissolve the examples into the explanations themselves then, as suggested here.

> I'm trying to reorganize first as a quick iteration

OK. Let's try to check we have a check box somewhere to follow up on this.